main.js

```diff
@@ -387,3 +395,75 @@ export function maxEncodedIdBytes({
   }
   return plainBytes + (multibase ? 1 : 0);
 }
+
+/**
+ * Generates a multibase seed.
+ *
+ * @returns {string} - A multibase seed.
+ */
+export async function generateMultibaseSeed() {
+  // 256 bit (32 byte) random id generator
+  const generator = new IdGenerator({
+    bitLength: SEED_BITS_SIZE
+  });
+    // generate a random seed
+  const seedBytes = await generator.generate();
+  if(seedBytes.length !== SEED_BYTE_SIZE) {
```

Comment:
Is this needed?

Reply:
+1, I don't think we need this check here.